CsharpExtrasTest/Extensions/StringExtensionTest.cs

```diff
@@ -136,7 +203,15 @@ public void SplitEmptyStrIsSingletonEmptyStr()
         [Category("Unit")]
         public void EqualsIgnoreDigitsTrueWhenBothSidesHaveDifferentDigits()
         {
-            Assert.IsTrue("s000df11asd1hf23las".EqualsIgnoreDigits("sdf33asdhf057la12389s"));
+            //Arrange
+            string value = "s000df11asd1hf23las";
+            string digits = "sdf33asdhf057la12389s";

```

Comment:
Looks like this variable is not used for anything. I think you may have missed replacing the string below with this variable.
The naming of the variables is also not very intuitive as both variables contain digits. Not quite sure what to name them, maybe `valueWithDigits1` and `valueWithDigits2` would make more sense? I'm open to suggestions though.

Reply:
Always in test cases I would name the variables expected(Something) and actual(Something). Usually the things you are comparing are an expected piece of data vs an actual piece of data that arose from some action.

Reply:
I agree, but in this case both of these variables are test inputs, not actual or expected results. The expected/actual result is a boolean.

Reply:
Aaaaah... Thank you for letting me see the light!!